app/backend/src/couchers/models.py

```diff
@@ -1581,8 +1591,8 @@ class UserBlock(Base):
     blocked_user_id = Column(ForeignKey("users.id"), nullable=False)
     time_blocked = Column(DateTime(timezone=True), nullable=False, server_default=func.now())
 
-    is_blocking_user = relationship("User", backref="is_blocking_user", foreign_keys="UserBlock.blocking_user_id")
-    is_blocked_user = relationship("User", backref="is_blocked_user", foreign_keys="UserBlock.blocked_user_id")
+    blocking_user = relationship("User", foreign_keys="UserBlock.blocking_user_id")
+    blocked_user = relationship("User", foreign_keys="UserBlock.blocked_user_id")
```

Comment:
Removed the backref, since it seems to me we don't use user.is_blocking/blocked_user and there's no real need for it (especially since it returns a list of all UserBlocks where the User is the blocking/blocked user, which is, IMO, rather confusing and unwieldly). Trivial to add it back in and reinstantiate the test I removed from test_blocking.py.